SPEC.md

```diff
@@ -208,8 +208,8 @@ Additional isolators will be added to this specification over time.
 |blockIO/writeBandwidth   |string|"&lt;path to file&gt; &lt;bytes&gt;"|"/tmp 1K"                           |
 |networkIO/readBandwidth  |string|"&lt;device name&gt; &lt;bytes&gt;" |"eth0 100M"                         |
 |networkIO/writeBandwidth |string|"&lt;device name&gt; &lt;bytes&gt;" |"eth0 100M"                         |
-|privateNetwork           |string|"&lt;true&#124;false&gt;"           |"true"                              |
-|capabilities/boundingSet |string|"&lt;cap&gt; &lt;cap&gt; ..."       |"CAP_NET_BIND_SERVICE CAP_SYS_ADMIN"|
+|private-network          |string|"&lt;true&#124;false&gt;"           |"true"                              |
```

Comment:
Good catch. Clearly isolators are an area that haven't received much love in a while since none of the names have been updated to the AC Name type! Would you mind changing the others too? (blockIO and networkIO)